ironfish-rust-nodejs/src/frost.rs

```diff
@@ -177,7 +176,7 @@ pub fn split_secret(
 
     Ok(TrustedDealerKeyPackages {
         verifying_key: bytes_to_hex(&t.verifying_key),
-        proof_generation_key: t.proof_generation_key.hex_key(),
+        proof_authorizing_key: bytes_to_hex(&t.proof_authorizing_key.to_bytes()),
```

Comment:
you can do proof_authorizing_key.hex_key() now I believe

Reply:
updates!